src/Splat.Autofac/AutofacDependencyResolver.cs

```diff
@@ -50,7 +50,7 @@ public AutofacDependencyResolver(ContainerBuilder builder)
         }
 
         /// <inheritdoc />
-        public virtual object? GetService(Type serviceType, string? contract = null)
+        public virtual object GetService(Type serviceType, string? contract = null)
```

Comment:
I would keep object? GetService rather than the non-nullable version.

Reply:
object? Will require more GetService<T?>()! when working with nullable types, perhaps try both options so you can see what I mean. If you register a nullable type you will either get the value or a null.
However I can see the other side where you want to return the reference to a registered item even if you registered it with a nullified nullable type.
The store can hold nullable types now if but not all our supported ones there's at least 1 that hasn't got nullable store's.
I think you are right, and although it might seem more hassle to add a ! It has more capability in the long run.

Reply:
object? GetService and 
IEnumerable<object?> GetServices wins